dehoister/dehoister.py

```diff
@@ -266,6 +264,53 @@ async def clean(self, ctx: commands.Context):
     async def _set(self, ctx: commands.Context):
         """Settings for Dehoister."""
 
+    @_set.group()
+    async def ignore(self, ctx: commands.Context):
+        """
+        Add and remove certain users from being ignored by the auto-dehoister.
+        """
```

Comment:
```suggestion
        """Add and remove certain users from being ignored by the auto-dehoister."""
```
Just for styling here